### PR TITLE
Upgrade Jackson to 2.16.2

### PR DESCRIPTION
**Pull request by hiroyuki-sato**

### CODE REVIEW

File: gradle/libs.versions.toml
Comment (dmikurube) [inline gradle/libs.versions.toml]:
```suggestion

```

**Review by dmikurube — APPROVED** [review]
LGTM.

It's okay this time, but let me leave a recommendation for next time -- when upgrading a dependency library, it'd be nice to have `gradle.lockfile` at first before upgrade, and then do the upgrade. By this order, we can confirm which library is upgraded explicitly.

**Comment by dmikurube** [comment]
jfyi, you're one of the maintainers of this plugin now. You can merge pull requests by yourself once approved. ;)

**Comment by hiroyuki-sato** [comment]
Thanks!

**Pull request merged**
